### PR TITLE
Feature/structure

### PR DESCRIPTION
**Pull request by tejue**
add general structure and first description.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est
+            Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur
+            sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore
+            et dolore magna aliquyam erat, sed diam voluptua. At vero eos et
+            accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren,
+            no sea takimata sanctus est Lorem ipsum dolor sit amet.
+          </figcaption>
+        </figure>
+      </section>
+      <section id="contac">
```
Comment (makitango) [inline index.html:34]: Maybe change "contac" to "contact"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>my personal website</title>
+  </head>
+  <body>
+    <header>
+      <h1>Daniella Teresa Jüngel</h1>
+      <nav id="nav">
+        <a href="Contact">Contact</a>
```
Comment (makitango) [inline index.html:13]: # + lowercass

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <a href="Biography">Biography</a>
```
Comment (makitango) [inline index.html:14]: # + lowercass

File: index.html (hunk 2 above)
Comment (josch87) [reply index.html:13]: not a valid link

File: index.html (hunk 3 above)
Comment (josch87) [reply index.html:14]: not a valid link

**Review by josch87 — COMMENTED** [review]
I go with the other comments

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            erat, sed diam voluptua. At vero eos et accusam et justo duo dolores
+            et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est
+            Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur
+            sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore
+            et dolore magna aliquyam erat, sed diam voluptua. At vero eos et
+            accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren,
+            no sea takimata sanctus est Lorem ipsum dolor sit amet.
+          </figcaption>
+        </figure>
+      </section>
```
Comment (DErdmann1) [inline index.html:33]: Nice idea to put it in a section!

**Review by DErdmann1 — COMMENTED** [review]
Looks good Teresa :)

**Pull request merged**